src/components/modules/renderer.ts

```diff
@@ -63,12 +65,14 @@ export default class Renderer extends Module {
    */
   public async insertBlock(item: OutputBlockData): Promise<void> {
     const { Tools, BlockManager } = this.Editor;
+    const id = item.id || _.generateUuidv4();
```

Comment:
```suggestion
    const id = item.id;
```

When `const id` is undefined, a new ID is generated in `BlockManager.insert()`.